### DISCUSSION TITLE
Location for thumbnails

### DISCUSSION THREAD

**Discussion by bluefuzz** [rern/rAudio#922] [General]
Is it possible to get rAudio to save thmbnails somewhere else than in the directory where the music is stored? Almost all my music on my NAS drive has embedded artwork and a folder.jpg in each album folder. Why does rAudio have to add a thumbnail and coverart file to the same folder? It clutters things ap and confuses other applications - and I would prefer to mount my Music drive read-only. Niether Moode nor Volumio seem to need to do this and AFAIK they use MPD too ...

**Comment by cdevalland** [comment]
Hi,
same issue. I prefer to mount my NAS drive as read-only.
Why not just to display folder.jpg in each folder ?
Although Thanks for this great job.

**Comment by rern** [comment]
Thumbnails are created for optimum quality and speed:
- Main cover arts should be as large as possible so they're always good in any scales: e.g. 1400 x 1400 - 1.4MB (can be any sizes)
- `coverart.jpg` - Thumbnails in Album mode and bookmarks: 200 x 200 - 70KB - display at 100 x 100
- `thumb.jpg` - Thumbnails in directory list: 80 x 80 - 15KB - display at 40 x 40

Those who prefer to display the main cover arts as thumbnails can manually symlink them in each directory. (Cover arts other than `jpg` must be converted.)

**Comment by cdevalland** [comment]
I finally gave write access to my music folder and I get all thumbnails.
Thanks.
